sound/soc/sof/intel/hda-dai.c

```diff
@@ -404,9 +414,22 @@ static int non_hda_dai_hw_params(struct snd_pcm_substream *substream,
 	dma_config->pre_allocated_by_host = 1;
 	dma_config->dma_channel_id = stream_id - 1;
 	dma_config->stream_id = stream_id;
-	dma_config->dma_stream_channel_map.device_count = 0; /* mapping not used */
+	/* Currently only 1 device for each DMA */
```

Comment:
You need to explain what a 'device' is in comments and the commit message. I don't know if this is a link or a PDI or something else.

The comment above is really not helping.